graphql/batch_executor.go

```diff
@@ -157,10 +157,11 @@ func executeBatchWorkUnit(unit *WorkUnit) []*WorkUnit {
 func executeNonExpensiveWorkUnit(unit *WorkUnit) []*WorkUnit {
 	results := make([]interface{}, 0, len(unit.sources))
 	for idx, src := range unit.sources {
-		if unit.objectName == "Query" {
-			unit.Ctx = context.WithValue(unit.Ctx, nonExpensive{}, struct{}{})
+		ctx := unit.Ctx
+		if unit.objectName != "Mutation" {
```

Comment:
did we still want to set the objectName "properly" everywhere.  I think right now it only gets set for the top-level "Mutation" and "Query" types.

Reply:
we can, I just wasn't sure if we needed to pass that information down

Reply:
we should add it back since right now objectName is not "really" what it says it is.  I'd rather get it done now instead of wondering why it wasn't set properly later.

Additionally can we add a comment here explaining what we're doing?